fluentd/run.sh

```diff
@@ -235,7 +235,21 @@ fi
 
 if [[ "${USE_REMOTE_SYSLOG:-}" = "true" ]] ; then
     # The symlink is a workaround for https://github.com/openshift/origin-aggregated-logging/issues/604
-    ln -s /opt/app-root/src/gems/fluent-plugin-remote-syslog*/lib/fluentd/plugin/*.rb /etc/fluent/plugin/
+    found=
+    for file in /usr/share/gems/gems/fluent-plugin-remote-syslog-*/lib/fluentd/plugin/*.rb ; do
+        if [ -f "$file" ] ; then
+            ln -s $file /etc/fluent/plugin/
+            found=true
+        fi
+    done
+    if [ -z "${found:-}" ] ; then
+        # not found in rpm location - look in alternate location
+        for file in /opt/app-root/src/gems/fluent-plugin-remote-syslog*/lib/fluentd/plugin/*.rb ; do
+            if [ -f "$file" ] ; then
+                ln -s $file /etc/fluent/plugin/
+            fi
+        done
+    fi
```

Comment:
Do we have some error report if the plugin does not exist in the both places?  

If it is guaranteed to be found at the either location, please ignore this comment. :)

Reply:
There are no guarantees in software :-)

The error report will be "no such plugin - remote_syslog" from fluentd

Reply:
I thought it could be guaranteed by the gem/rpm dependency...  But the error message from fluentd looks clear enough. :)

/lgtm